src/components/refreshdialog/refreshdialog.js

```diff
@@ -71,14 +76,15 @@ function onSubmit(e) {
 
     const mode = dlg.querySelector('#selectMetadataRefreshMode').value === 'scan' ? 'Default' : 'FullRefresh';
     const replaceAllImages = mode === 'FullRefresh' && dlg.querySelector('.chkReplaceImages').checked;
+    const replaceTrickplayImages = mode === 'FullRefresh' && dlg.querySelector('.chkReplaceTrickplayImages').checked;
```

Comment:
It might be a good idea to allow trickplay images to be regenerated without refreshing metadata.

Reply:
Oops, sorry.
`mode === 'FullRefresh'` doesn't mean "full refresh" (as I thought), it means "not scan for new".
`chkReplaceTrickplayImages` is currently hidden when "scan", so we still need that condition.